ingenialink/canopen/servo.py

```diff
@@ -20,130 +15,6 @@
-    )
-}
-
-CONTROL_WORD_REGISTERS = {
-    1: CanopenRegister(
-        identifier='', units='', subnode=1, idx=0x2010, subidx=0x00,
-        cyclic='CYCLIC_RX', dtype=REG_DTYPE.U16, access=REG_ACCESS.RW
-    ),
-    2: CanopenRegister(
-        identifier='', units='', subnode=2, idx=0x2810, subidx=0x00,
-        cyclic='CYCLIC_RX', dtype=REG_DTYPE.U16, access=REG_ACCESS.RW
-    ),
-    3: CanopenRegister(
-        identifier='', units='', subnode=3, idx=0x3010, subidx=0x00,
-        cyclic='CYCLIC_RX', dtype=REG_DTYPE.U16, access=REG_ACCESS.RW
-    )
-}
-
-STORE_COCO_ALL = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x1010, subidx=0x01, cyclic='CONFIG',
-    dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-)
-
-RESTORE_COCO_ALL = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x1011, subidx=0x01, cyclic='CONFIG',
-    dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-)
-
-STORE_MOCO_ALL_REGISTERS = {
-    1: CanopenRegister(
-        identifier='', units='', subnode=1, idx=0x26DB, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    ),
-    2: CanopenRegister(
-        identifier='', units='', subnode=2, idx=0x2EDB, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    ),
-    3: CanopenRegister(
-        identifier='', units='', subnode=3, idx=0x36DB, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    )
-}
-
-RESTORE_MOCO_ALL_REGISTERS = {
-    1: CanopenRegister(
-        identifier='', units='', subnode=1, idx=0x26DC, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    ),
-    2: CanopenRegister(
-        identifier='', units='', subnode=2, idx=0x2EDC, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    ),
-    3: CanopenRegister(
-        identifier='', units='', subnode=3, idx=0x36DC, subidx=0x00,
-        cyclic='CONFIG', dtype=REG_DTYPE.U32, access=REG_ACCESS.RW
-    )
-}
-
-MONITORING_DIST_ENABLE = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x58C0, subidx=0x00, cyclic='CONFIG',
-    dtype=REG_DTYPE.U16, access=REG_ACCESS.RW
-)
-
-MONITORING_NUMBER_MAPPED_REGISTERS = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x58E3, subidx=0x00, cyclic='CONFIG',
-    dtype=REG_DTYPE.U16, access=REG_ACCESS.RW
-)
-
-MONITORING_REMOVE_DATA = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x58EA, subidx=0x00, cyclic='CONFIG',
-    dtype=REG_DTYPE.U16, access=REG_ACCESS.WO
-)
-
-MONITORING_BYTES_PER_BLOCK = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x58E4, subidx=0x00, cyclic='CONFIG',
-    dtype=REG_DTYPE.U16, access=REG_ACCESS.RO
-)
-
-MONITORING_ACTUAL_NUMBER_BYTES = CanopenRegister(
-    identifier='', units='', subnode=0, idx=0x58B7, subidx=0x00, cyclic='CONFIG',
-    dtype=REG_DTYPE.U32, access=REG_ACCESS.RO
-)
-
 MONITORING_DATA = CanopenRegister(
```

Comment:
Why you didn't move MONITORING_DATA, MONITORING_DISTURBANCE_VERSION, and DIST_DATA registers inside the class?

Reply:
Because it was not necessary. For coherence, I will move all the registers within the class.